server/lists/adapters/actions/externalInterestsSummaryAdapter.js

```diff
@@ -0,0 +1,12 @@
+module.exports = async (actions) => {
+
+    const { recordInterest: interests } = actions.caseActionData;
+
+    return await Promise.all(interests.map( async interestEntry => {
```

Comment:
This breaches Sonarlint rule: "await" should not be used redundantly

An async function always wraps the return value in a Promise. Using return await is therefore redundant.